app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -3,6 +3,7 @@ import type { ReactNode } from 'react';
 import useObjectMemo from '@18f/identity-react-hooks/use-object-memo';
 import AnalyticsContext from './analytics';
 import DeviceContext from './device';
+import FeatureFlagContext from './feature-flag';
```

Comment:
same as above re: which context is needed (hopefully this isn't annoying - I thought it might be a helpful marker)

Reply:
✅ [link](https://github.com/18F/identity-idp/pull/10156#discussion_r1517894124)